NAMESPACE

```diff
@@ -13,7 +13,7 @@ export(stabsel, stabsel.matrix, stabsel.data.frame, stabsel.stabsel,
        selected, selected.stabsel,
        plot.stabsel, print.stabsel, print.stabsel_parameters,
        glmnet.lasso, glmnet.lasso_maxCoef, lars.lasso, lars.stepwise,
-       subsample)
+       subsample, stabs.quic, getLamPath)
```

Comment:
please use standard naming convention, i.e., package name followed by type of model, e.g., `quic.graphical_model` and do not export `getLamPath` (I do not see that the user needs that.